lib/page-label/index.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import box from '../box/index';
+
+var pageLabel = angular.module('guPageLabel', ['guBox']);
+
+pageLabel.directive('guPageLabel', function pageLabelDirective(){
+  return {
+    restrict: 'E',
+    transclude: true,
+    replace: true,
+    template: (el, attrs) => `<gu-box variant="tertiary" class="page-label" ng-class="{ open: isOpen }" ng-mouseover="isOpen = true", ng-mouseleave="isOpen = false">
```

Comment:
To avoid polluting the scope with `isOpen`, you should make this an isolate scope.